dcos_installer/backend.py

```diff
@@ -145,7 +145,14 @@ def calculate_aws_template_storage_region_name(
 
     try:
         location_info = session.client('s3').get_bucket_location(Bucket=aws_template_storage_bucket)
-        return location_info["LocationConstraint"]
+        loc = location_info["LocationConstraint"]
+        if loc:
```

Comment:
Oh, in that case, shouldn't it be `if loc is None` ?

This review makes the case that, even after reading the description, I could not  get what is being checked.